src/components/Shop/index.js

```diff
@@ -9,17 +9,23 @@ const Shop = ({ shop, distance }) => {
   const formatName = () => {
     const { chain, title } = shop;
 
+    // If title ends up as null, just return the chain
+    if (title === null) {
+      return `${chain}`;
+    }
+
     if (title.includes('@')) {
       return title;
     }
 
     return `${chain} @ ${title}`;
   };
+
   function openMapApp(shopCoordinates) {
-    const isAppleOS = navigator.platform.match(/(iPhone|iPod|iPad)/i)?true:false;
+    const isAppleOS = navigator.platform.match(/(iPhone|iPod|iPad)/i);
 
     // Open Apple Maps if OS is iOS
-    if (isAppleOS) {
+    if (isAppleOS != null) {
```

Comment:
could you use triple equals? `===`

Alternatively I think doing 
```js
/(iPhone|iPod|iPad)/i.test(navigator.platform)
```
should return a boolean